modules/catalog/src/testFixtures/java/org/apache/ignite/internal/catalog/BaseCatalogManagerTest.java

```diff
@@ -81,4 +91,66 @@ public void tearDown() throws Exception {
                 .map(component -> component::stop)
         );
     }
+
+    protected static CreateHashIndexParams createHashIndexParams(
+            String indexName,
+            boolean uniq,
+            @Nullable List<String> indexColumns
+    ) {
+        CreateHashIndexParams.Builder builder = CreateHashIndexParams.builder()
+                .schemaName(DEFAULT_SCHEMA_NAME)
+                .tableName(TABLE_NAME)
+                .indexName(indexName);
+
+        if (uniq) {
+            builder.unique();
+        }
+
+        if (indexColumns != null) {
+            builder.columns(indexColumns);
+        }
+
+        return builder.build();
+    }
+
+    protected static CreateHashIndexParams createHashIndexParams(
+            String indexName,
+            @Nullable List<String> indexColumns
+    ) {
+        return createHashIndexParams(indexName, false, indexColumns);
+    }
+
+    protected static CreateSortedIndexParams createSortedIndexParams(
+            String indexName,
+            boolean uniq,
+            @Nullable List<String> indexColumns,
+            @Nullable List<CatalogColumnCollation> columnsCollations
+    ) {
+        CreateSortedIndexParams.Builder builder = CreateSortedIndexParams.builder()
+                .schemaName(DEFAULT_SCHEMA_NAME)
+                .tableName(TABLE_NAME)
+                .indexName(indexName);
+
+        if (uniq) {
+            builder.unique();
+        }
```

Comment:
`unique()` might accept the argument to avoid unwanted `if`

Reply:
fix it